src/lib-components/NsTimePicker.vue

```diff
@@ -55,6 +55,10 @@ export default {
       type: String,
       default: "",
     },
+    dropDirection: {
+      type: String,
+      default: "down"
+    }
```

Comment:
Let's add a validator to ensure that `up| down | auto` is passed to the component. Example: https://github.com/NethServer/ns8-ui-lib/blob/main/src/lib-components/NsComboBox.vue#L254

Reply:
I noticed an error in the example I linked above :scream: 

This is correct istead: https://github.com/NethServer/ns8-ui-lib/blob/main/src/lib-components/NsComboBox.vue#L249

After fixing `dropDirection` validator, could you also fix NsComboBox `tooltipDirection` validator in a separate commit please?

Reply:
Separate commit in another pr or here?

Reply:
Here is fine